src/routes/CheckIn/components/CheckInForm.js

```diff
@@ -0,0 +1,163 @@
+      dateOfBirth,
+      phoneNumber,
+      emailAddress,
+
+      gender,
+      isTransexual,
+      isLGBTQ,
+
+      ethnicity,
+
+      language,
+
+      hasBeenInFosterCare,
+
+      hasServedInTheMilitary,
+
+      primaryHealthcareLocation,
+
+      isHomeless,
+      lengthOfHomelessness,
+    },
+    resetForm,
+    handleSubmit,
+    submitting,
+    updatePrimaryInfo,
+  } = props
+
+  const _onSubmit = () => {
+    props.updatePrimaryInfo(props.fields)
+  }
+
+  return (
+    <form
+      className={classes.checkInForm}
+      onSubmit={handleSubmit(_onSubmit)}>
+      <div>
+        <label>First Name</label>
+        <input type="text" placeholder="First Name" {...firstName} />
+        {firstName.touched && firstName.error && <div>{firstName.error}</div>}
+      </div>
+
+      <div>
+        <label>Last Name</label>
+        <input type="text" placeholder="Last Name" {...lastName} />
+        {lastName.touched && lastName.error && <div>{lastName.error}</div>}
+      </div>
+
+      <div>
+        <label>SSN</label>
+        <input type="phone" placeholder="xxx-xx-xxxx" {...socialSecurityNumber} />
+        {socialSecurityNumber.touched && socialSecurityNumber.error && <div>{socialSecurityNumber.error}</div>}
+      </div>
+
+      <div>
+        <label>Date of Birth</label>
+        <input type="text" placeholder="xx-xx-xxxx" {...dateOfBirth} />
+        {dateOfBirth.touched && dateOfBirth.error && <div>{dateOfBirth.error}</div>}
+      </div>
+
+      <div>
+        <label>Phone</label>
+        <input type="text" placeholder="xxx-xxx-xxxx" {...phoneNumber} />
+        {phoneNumber.touched && phoneNumber.error && <div>{phoneNumber.error}</div>}
+      </div>
+
+      <div>
+        <label>Email Address</label>
+        <input type="email" placeholder="johndoe@google.com" {...emailAddress} />
+        {emailAddress.touched && emailAddress.error && <div>{emailAddress.error}</div>}
+      </div>
+
+      <div>
+        <label>Gender</label>
+        <input type="radio" {...gender} />Male
+        <input type="radio" {...gender} />Female
+        <input type="checkbox" {...isTransexual} />Different from birth gender?
+        {gender.touched && gender.error && <div>{gender.error}</div>}
+      </div>
+
+      <div>
+        <label>Sexuality</label>
+        <input type="checkbox" {...isLGBTQ} />LGBTQ?
+      </div>
+
+      <div>
+        <label>Ethnicity</label>
+        <input type="checkbox" {...ethnicity} />African American
+        <input type="checkbox" {...ethnicity} />Asian / Pacific Islander
+        <input type="checkbox" {...ethnicity} />Caucasian
+        <input type="checkbox" {...ethnicity} />Latino
+        <input type="checkbox" {...ethnicity} />Native American
+        Other <input type="text" {...ethnicity} />
+        {ethnicity.touched && ethnicity.error && <div>{ethnicity.error}</div>}
+      </div>
+
+      <div>
+        <label>Primary Language</label>
+        <input type="checkbox" {...language} />English
```

Comment:
`{...language}` destructures the values of `language` into the `input` field.  In this case, `languages` is actually an object (from `redux-form`) which has more than just the value itself -- it has methods like `onChange/onBlur` and attributes like `valid/dirty/value` which help it maintain its own state.